runners/main_runner.py

```diff
@@ -0,0 +1,123 @@
+from dataclasses import dataclass
+from itertools import product
+from pathlib import Path
+import warnings
+import yaml
+
+from misc.net_loader import load_network
+from misc.utils import *
+from tqdm import tqdm
```

Comment:
Consider using `from tqdm.auto import tqdm`. This line will automatically import way better looking progress bar when used inside of jupyter

Reply:
Yes, I know that :) However I'm using tqdm outside Jupter, in fact in the runner which is a classis python script.